tensorflow/core/kernels/sparse_reorder_op.cc

```diff
@@ -27,6 +27,7 @@ limitations under the License.
 #include "tensorflow/core/framework/types.h"
 #include "tensorflow/core/lib/gtl/inlined_vector.h"
 #include "tensorflow/core/util/sparse/sparse_tensor.h"
+#include "tensorflow/core/util/overflow.h"
```

Comment:
```suggestion
```

This is no longer needed

Reply:
@mihaimaruseac Thanks. The extraneous include has been removed.